app/javascript/admin/controllers/config_controller.js

```diff
@@ -246,4 +259,68 @@ export default class ConfigController extends Controller {
     }
   }
   // AUTH PROVIDERS FUNCTIONS END
+
+  enabledProvidersWithMissingKeys() {
+    const providersWithMissingKeys = [];
+    document
+      .querySelectorAll('[data-enable-auth="true"]')
+      .forEach((provider) => {
+        const providerName = provider.dataset.authProviderEnable;
+        const officialName = provider.dataset.authProviderOfficial;
+        if (
+          document.getElementById(`site_config_${providerName}_key`).value
+            .length === 0 ||
+          document.getElementById(`site_config_${providerName}_secret`).value
+            .length === 0
+        ) {
```

Comment:
I _believe_ that if the element doesn't have a value, it should be an empty string (`""`). Empty strings are falsy in JavaScript, so I think you don't need to do the `.length === 0` check here